packages/amplify-graphql-model-transformer/package.json

```diff
@@ -21,7 +21,7 @@
     "access": "public"
   },
   "scripts": {
-    "build": "tsc",
+    "build": "tsc && cd rds-lambda && mkdir -p node_modules && rm -rf node_modules && npm install && tsc && cp -r node_modules lib && cd lib && bestzip --force node ../../lib/rds-lambda.zip ./*",
```

Comment:
do we still need this?

Reply:
This is going to change in the upcoming PR as we are moving this logic to lambda layer. This is the current state.